butterfly-utilities/src/main/java/com/paypal/butterfly/utilities/xml/XmlXPathExists.java

```diff
@@ -0,0 +1,114 @@
+package com.paypal.butterfly.utilities.xml;
```

Comment:
Move this class to package `utilities.conditions`. See my other comment below to know why.